tutorials/international/index.md

```diff
@@ -25,6 +25,56 @@ supports. Flutter provides widgets and classes that help with
 internationalization and the Flutter libraries themselves are
 internationalized.
 
+### Setting up an Internationalized app: the `flutter_localizations` package
+
+By default Flutter only provides US English localizations. To add
+support for other languages, applications must specify some additional
+`MaterialApp` properties and include a separate package called
+`flutter_localizations`.  This package currently supports about 15
+languages with more to come.
+
+International apps must add a `pubspec.yaml` dependency on flutter_localizations:
```

Comment:
maybe better as "To use flutter_localizations, add the package a dependency to your pubspec.yaml file"